docs/source/tutorial/basic.rst

```diff
@@ -417,9 +416,9 @@ just replace :func:`~serializers.save_npz` and :func:`~serializers.load_npz` by
 
 Example: Multi-layer Perceptron on MNIST
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Now you can solve a multiclass classification task using a multi-layer perceptron.
-We use hand-written digits dataset called `MNIST <http://yann.lecun.com/exdb/mnist/>`_, which is one of the long-standing de facto "hello world" of machine learning.
++
```

Comment:
Remove this redundant bullet

https://github.com/rcalland/chainer/blob/master/docs/source/tutorial/basic.rst#example-multi-layer-perceptron-on-mnist